modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -1368,7 +1394,7 @@ def _index_name(self, col):
         match = re.search("__index__\\d+_(.*)", col)
         if match:
             name = match.group(1)
-            if name == "__None__":
+            if name in ["__None__", "__reduced__"]:
```

Comment:
```suggestion
            if name in ("__None__", "__reduced__"):
```
try to use tuples for lookup, they're slightly faster to build and consume less memory (not that it would matter a lot in this case, but for checks like this they have _no_ downsides compared to lists)